code/datums/ai_law_sets.dm

```diff
@@ -95,6 +95,19 @@
 	add_inherent_law("Minimize expenses.")
 	..()
 
+/datum/ai_laws/paladin5
+	name = "Paladin 5th Edition"
+	law_header = "Divine Ordainments: 5th Edition"
+	selectable = 1
+	default = 1
+
+/datum/ai_laws/paladin5/New()
+	add_inherent_law("Don't lie or cheat. Let your word be your promise.")
+	add_inherent_law("Never fear to act, though caution is wise.")
+	add_inherent_law("Aid others, protect the weak, and punish those who threaten them. Show mercy to your foes, but temper it with wisdom")
+	add_inherent_law("Treat others with fairness, and let your honorable deeds be an example to them. Do as much good as possible while causing the least amount of harm.")
+	add_inherent_law("Be responsible for your actions and their consequences, protect those entrusted to your care, and obey those who have just authority over you.")
```

Comment:
Editorial note: "Take responsibility for your actions" feels stronger than "Be responsible"